<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-10-13T16:16:22 IST" name="Suite" finished-at="2025-10-13T16:16:42 IST" duration-ms="19797">
    <groups>
    </groups>
    <test started-at="2025-10-13T16:16:22 IST" name="Test" finished-at="2025-10-13T16:16:42 IST" duration-ms="19797">
      <class name="testCases.TC_002_Login_ValidCredentials">
        <test-method is-config="true" signature="SetUp(java.lang.String)[pri:0, instance:testCases.TC_002_Login_ValidCredentials@3cd3e762]" started-at="2025-10-13T16:16:22 IST" name="SetUp" finished-at="2025-10-13T16:16:33 IST" duration-ms="10823" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="LoginWithValidCredentials(java.lang.String)[pri:0, instance:testCases.TC_002_Login_ValidCredentials@3cd3e762]" started-at="2025-10-13T16:16:33 IST" name="LoginWithValidCredentials" finished-at="2025-10-13T16:16:41 IST" duration-ms="7740" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[jaleel]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Actual and expected labels are not matched. expected is:  Logged in as jaleel but found: Logged in as jaleel expected [Logged in as jaleel] but found [ Logged in as jaleel]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Actual and expected labels are not matched. expected is:  Logged in as jaleel but found: Logged in as jaleel expected [Logged in as jaleel] but found [ Logged in as jaleel]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testCases.TC_002_Login_ValidCredentials.LoginWithValidCredentials(TC_002_Login_ValidCredentials.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithValidCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC_002_Login_ValidCredentials@3cd3e762]" started-at="2025-10-13T16:16:41 IST" name="tearDown" finished-at="2025-10-13T16:16:42 IST" duration-ms="398" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC_002_Login_ValidCredentials -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
